commands/build/README.md

```diff
@@ -24,3 +24,117 @@ Options:
   --mode           Explicitly set mode
                                  [string] [choices: "production", "development"]
 ```
+
+## Example
+
+### Configuration file
+
+Build the bundle with a nebula configuration json file in a new path
+
+```sh
+nebula build --config ../../nebula.config.js
```

Comment:
the '.../.../' makes no sense in this example.
Also, nebula.config.js is the default value of this, so change the example to 'my-nebula-config.js' to differentiate.

Reply:
good point. Thanks. fixed.